be/src/pipeline/exec/scan_operator.cpp

```diff
@@ -1456,36 +1450,19 @@ Status ScanOperatorX<LocalStateType>::get_block(RuntimeState* state, vectorized:
     }};
 
     if (state->is_cancelled()) {
```

Comment:
And  also should check if scanner ctx is nullptr here, if null ptr should cancel.
sometimes the scanner ctx is not inited successully. and it will core if not check it.